### DISCUSSION TITLE
[Grant] Metaplex Whitepaper NFT Incentive Program

### DISCUSSION THREAD

**Discussion by MetaplexFoundation** [metaplex-foundation/dao#6] [Community Grant Submissions]
**Submission details**
Submitter: Charlesfoodog
Email: [charlessublette@gmail.com](mailto:charlessublette@gmail.com)
Submitter wallet: 2YxHRtQJ914rQkxqyFYC1VxfpJWMZHLqc7SAMPVjipRv
Beneficiary wallet: Gf2fRncE8Kws3pt8KYwfvAMi4rB526j8cme7zUztrnhL
$MPLX requested: 12,050,000

NOTE: All $MPLX delivered via DAO grants will be locked for 1 year.

**Team**

Abrar Musa. Lead Developer:

10 years of code development.
3 Years of development inside Solana. Has contracted work on multiple snapshot contracts for NFT drops.
Senior Web Developer
BA in Computer Sci and Economics.

Charles Sublette:
Marketing, Sales, and Project Manager in Web/Dev sphere for 10 years.
Solana Maxi Since 2021.

I will facilitate the management of the developer by constructing the right procedures and outlining the development to fit the correct timeline.

**Abstract**
These tokens distribution will be to incentivize long-term holders of the Metaplex Whitepaper. This will encourage community growth, engagement, loyalty, and marketing inside the Metaplex ecosystem. As well as add another level of decentralization of the token.

**Description**
This proposal is to allocate 12,000,000 Metaplex tokens from the Metaplex DAO treasury to Metaplex Whitepaper Holders. 12000 tokens per NFT holder Will be given 1000 MPLX per month.(1000 NFTs exist in the collection in total) for 12 months. Metaplex NFT holders will get access to a special exclusive chat section in the Metaplex discord.

Starting distributions will begin August 1st, 2024, and will end August 1st, 2025. Each month a snapshot will be taken of current holders. This will occur at a randomly picked time inside each monthly block. Distributions will also be at the end of every month.

(50,000) Tokens will be used for the development cost and maintenance of the smart contract to airdrop the tokens every month to Metaplex NFT holders (https://magiceden.io/marketplace/metaplex_whitepaper).

**Milestones (optional)**
As soon as the contract is approved I will contact you and start to block out time for Abrar to begin to work on the contract.

Here's a rough timeline if approval is done in two weeks.

Deposit 50,000 MPLX tokens into the wallet for development costs.

May 15th Start to build out the smart contract outline for scanning wallets for NFT holders.

May 25th First development of code.

June 1st Submit a stable build of airdrop and snapshot mechanism.

June 15th. Testing/ audit of the smart contract.

June 30th Push final code base to git repo for Metaplex team to audit.

July 15th Make an Announcement on Metaplex's official Twitter about the NFT Incentive program.

July 30th Push 12,000,000 million Metaplex tokens into a cold wallet for the airdrop. facilicated Metaplex.

August 1st, Begin Distribution of the code and airdrop tokens over the 12-month vesting.

Jan 1st 2024 6-month check of code audit

August 1st 2025. The last airdrop and the snapshot is taken. Final tokens are released.

Any feedback is appreciated. I am submitting this project because I want to see Metaplex and its community grow. This incentive will generate a lot of PR and help bring a stronger community that will expose the Metaplex Brand to a wider audience.

**Comment by MetaplexFoundation** [comment]
Voting for this grant proposal is now open for a total of 14 days. Share your thoughts here or on the DAO message board. 
https://app.realms.today/dao/Metaplex/proposal/9xGFiwQQV42ehtahp6U984Um4kRmCnfzMg2g3eRfGZKa

Voting will close April 29th.

**Comment by ozonelabs-io** [comment]
Yeah i think as 12 Million isnt big supply and there isnt any drops to whitepaper holder from long time. i think it will be great to have some drops to Them
